lib/kitchen/driver/vcenter.rb

```diff
@@ -80,6 +80,10 @@ class Vcenter < Kitchen::Driver::Base
         The 'aggressive_password' setting was renamed to 'vm_password' and will
         be removed in future versions.
       MSG
+      deprecate_config_for :customize, Util.outdent!(<<-MSG)
+        The `customize` setting was renamed to `vm_customization` and will
+        be removed in future versions.
+      MSG
```

Comment:
If a user has a `customize` section, it will now be ignored. Could we detect that and use the settings if present?

Reply:
I totally forgot that I did that with some other settings in the `save_and_verify_parameters` method. Added the same for `customize` and verified it'll both throw a warning and do the VM customization anyway.